.github/workflows/test.yml

```
@@ -0,0 +1,40 @@
+        node-version: '18'
+        cache: 'yarn'
+    
+    - name: Install dependencies
+      run: yarn install --frozen-lockfile
+    
+    - name: Type check
+      run: yarn tsc --noEmit
+
+    - name: build
+      run: yarn build
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)